packages/account/src/Sections/Security/ClosingAccount/closing-account-reason.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { Redirect } from 'react-router-dom';
+import { Loading, Modal, Text } from '@deriv/components';
+import { routes } from '@deriv/shared';
+import { useCloseDerivAccount } from '@deriv/api';
+import { Localize } from '@deriv/translations';
+import ClosingAccountHasPendingConditions from './closing-account-pending-conditions/closing-account-has-pending-conditions';
+import ClosingAccountReasonForm from './closing-account-reason-form';
+import ClosingAccountWarningModal from './closing-account-warning-modal';
+import ClosingAccountGeneralErrorContent from './closing-account-general-error-content';
+
+type TClosingAccountReasonProps = {
+    redirectToSteps: () => void;
+};
+
+const MAX_ALLOWED_REASONS = 3;
```

Comment:
I see certain constant values are being repeated, can we move them to a constant/config file 🙏🏼